test/integration/istio_http_integration_test.cc

```diff
@@ -249,6 +254,22 @@ class IstioHttpIntegrationTest : public HttpProtocolIntegrationTest {
     cleanupConnection(policy_connection_);
   }
 
+  ConfigHelper::ConfigModifierFunction addNodeMetadata() {
+    return [](envoy::config::bootstrap::v2::Bootstrap& bootstrap) {
+      ::google::protobuf::Struct meta;
+      MessageUtil::loadFromJson(
+          fmt::sprintf(R"({
+        "ISTIO_VERSION": "1.0.1",
```

Comment:
Do we have test to verify these local attributes ?

Reply:
Yes. direct mixer attributes are now empty, and the integration test relies on attributes from bootstrap to work correctly.